Multimedia keys improvements.

MaartenBent opened the pull request:
Follow up for PR #157.
- Simplified mapping keys on Windows.
- ~~Added key names to accelerator table (based on `gdk/keynames.txt`)~~
  - ~~What to do with `back`? It is also used for backspace. Maybe add `Browser` to the names?~~
  - ~~Does `\src\gtk\menu.cpp GetGtkHotKey` need to be updated?~~
- Handle keys in richtextctrl.
  - Using media keys resulted in tabs(?) being inserted.
- Support the keys in the samples that print pressed keys.

vadz commented on src/common/accelcmn.cpp:
```diff
@@ -112,6 +112,23 @@ static const struct wxKeyName
     { WXK_WINDOWS_RIGHT,    /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Windows_Right") },
     { WXK_WINDOWS_MENU,     /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Windows_Menu") },
     { WXK_COMMAND,          /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Command") },
+    { WXK_BROWSER_BACK,     /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Back") },
```
This one definitely needs to be changed to something else, we can't have non unique strings here.

Actually I also wonder if we need to have these keys here at all, I don't think I've ever seen them explicitly listed as accelerators. The point of the special keys is that they have a defined function, so does it really make sense to list them here? Either they do what is written on them, in which case it's useless, or it doesn't, in which case it's very confusing and shouldn't be encouraged.

MaartenBent replied:
I have never seen menu items with browser/media keys as accelerator either. I will remove it from the PR.

MaartenBent committed:
Multimedia keys improvements.
Simplified mapping keys on Windows.
Handle multimedia keys in the keyboard, text and treetest samples.
Ignore multimedia keys in wxRichTextCtrl.

vadz commented:
Thanks, committed as 3b98570f5780afba87d829e92b0d1e7ebdd6650e
